etna/media/blocks.py

```diff
@@ -3,34 +3,23 @@
 from django.template.loader import render_to_string
 
 from wagtail.core import blocks
-from wagtailmedia.blocks import ChooserBlock, AbstractMediaChooserBlock
+from wagtail.images.blocks import ImageChooserBlock
+from wagtailmedia.blocks import AbstractMediaChooserBlock
 
 
 class EtnaMediaBlock(AbstractMediaChooserBlock):
```

Comment:
Would you mind renaming to `MediaChooserBlock`?

Reply:
Done